cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -74,6 +74,18 @@ object CqlParser extends JavaTokenParsers
 
     str | blob | uuid | floats | int | boolean | nullConst
   }
+
+  def comment: Parser[Comment] = {
+    import Comment._
+
+    def str = "[0-9a-fA-F]".r
+    def dash = s"--$str".r ^^ SingleLineDash
+    def slash = s"//$str".r ^^ SingleLineSlash
+    def multi = s"/*$str*/".r ^^ MultiLine
```

Comment:
This won't work, the regex won't allow new lines.
Maybe write a test case first